WcaOnRails/app/models/registration.rb

```diff
@@ -25,21 +27,11 @@ def pending?
     end
   end
 
-  before_save :normalize_event_ids
   private def normalize_event_ids
     self.eventIds = events.map(&:id).join(" ")
   end
 
-  attr_writer :birthday
-  def birthday
-    birthYear == 0 || birthMonth == 0 || birthDay == 0 ? nil : Date.parse("%04i-%02i-%02i" % [ birthYear, birthMonth, birthDay ])
-  end
-
-  before_validation :unpack_dates
   private def unpack_dates
-    if @birthday.nil? && !birthday.blank?
```

Comment:
Ah, that's right. I wrote tests for this paradigm over in the user.rb
model, but didn't port them over to the Preregs model specs.

On Wed, Aug 26, 2015 at 3:54 PM, Tim Habermaas <notifications@github.com>
wrote:

> In WcaOnRails/app/models/registration.rb
> <https://github.com/cubing/worldcubeassociation.org/pull/141#discussion_r38045289>
> :
>
> >    private def unpack_dates
> > -    if @birthday.nil? && !birthday.blank?
>
> It looks like they wouldn't. Everything's green :/.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cubing/worldcubeassociation.org/pull/141/files#r38045289>
> .
>